src/Forerkort/Skjema.elm

```diff
@@ -0,0 +1,256 @@
+module Forerkort.Skjema exposing
```

Comment:
Du kan godt skrive inn kommentarer som overskrifter i denne filen for å bryte opp litt. Du kan se hvordan det er gjort i de andre skjema-filene.